src/Npgsql/Npgsql.csproj

```diff
@@ -20,6 +20,7 @@
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.3" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
+	<PackageReference Include="Microsoft.Bcl.HashCode" Version="1.1.0" />
```

Comment:
nit: indentation.

Reply:
Tabulation by VS. Visible only in mails and here. Will fix.